src/events/crawler/lib/fetch.js

```diff
@@ -1,107 +1,20 @@
 import cheerio from 'cheerio';
-import needle from 'needle';
 import csvParse from 'csv-parse';
-import puppeteer from 'puppeteer';
 import { PdfReader } from 'pdfreader';
-
-import * as datetime from './datetime.js';
+import puppeteer from 'puppeteer';
 import * as caching from './caching.js';
+import * as datetime from './datetime.js';
+import { get } from './get.js';
+
+export const fetch = get;
```

Comment:
NIT: a small comment to explain why we have to do this.

Reply:
👍